src/v3/src/util/locUtil.ts

```diff
@@ -17,12 +17,13 @@ export const loc = (
   key: string,
   bundleName?: string,
   params?: Array<string | number | boolean | unknown>,
+  replacerFn?: (content: string) => string,
```

Comment:
This function is used to replace tokens like `<$1></$1>` with an actual HTML element. The reason for this is because the parser removes these unknown tags.